src/Driver/Encodings.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Driver;
+
+final class Encodings
+{
+    /**
+     * Represents whether a value should be bound with ASCII encoding
+     *
+     * @see \PDO::PARAM_STR_CHAR
+     */
+    public const ASCII = 536870912;
```

Comment:
Since there's only one possible value of the encoding (UTF8 will be used implicitly), would it make sense to just introduce another parameter type? See https://github.com/doctrine/dbal/pull/3149 for the reference.